chart/values.yaml

```diff
@@ -304,6 +304,16 @@ etcd:
   # Ignored if lifecycleHooks is set or replicaCount=1
   removeMemberOnContainerTermination: false
 
+  # AutoCompaction
+  # Since etcd keeps an exact history of its keyspace, this history should be
+  # periodically compacted to avoid performance degradation
+  # and eventual storage space exhaustion.
+  autoCompactionMode: revision
+  autoCompactionRetention: 100
+  extraEnvVars:
+    - name: ETCD_QUOTA_BACKEND_BYTES
+      value: "8589934592"
```

Comment:
A comment with a human readable number would help here.

Reply:
Indeed, and I've added a few more comments as well.